tree/30_generic_methods/file_copy_from_remote_source_recursion.cf

```diff
@@ -71,6 +71,12 @@ bundle agent file_copy_from_remote_source_recursion(source, destination, recursi
         classes      => classes_generic_two("${old_class_prefix}", "${class_prefix}");
 
   methods:
+    iteration_2::
```

Comment:
can you add a comment on why adding this condition ? I'm pretty sure future us won't remember why the hell we define and error class